docs/CONTRIBUTING.md

```diff
@@ -40,6 +40,26 @@ An easier solution would be to use Docker.
 1. Run `npm test`
 1. IF eslint detect some issues run `npm run eslint-fix` before manually fixing the issue (Will save you time :smile:) and then run `npm test` again.
 
+### Common Errors
+**Linux**
+
+If you encounter this error when running redis-server:
+```
+# Could not create server TCP listening socket *:6379: bind: Address already in use
+```
+1. Run `sudo systemctl status redis` to check if the service is running
+2. Stop it with `sudo systemctl stop redis`.
+If this doesn't work, you can check and manually kill the instance using port 6379 by running
+`ps aux | grep redis`
+
+```
+user     2570  0.0  0.0 253100  4908 pts/1    Sl+  13:11   0:00 redis-server *:6379
+user     2588  0.0  0.0 215744   888 pts/0    S+   13:16   0:00 grep --color=auto redis
+```
+Run `sudo kill -9 2570`
```

Comment:
Again, this is unnecessary. See my comment above. FYI the process ID that you have added will likely not be the same for others running the same process. Suggesting an uninformed user to execute ```kill -9``` with ```sudo``` on a specific pid like you are suggesting here is dangerous and could cause some major issues on their machine.